wodles/aws/aws_s3.py

```diff
@@ -513,8 +511,15 @@ def send_msg(self, msg, dump_json=True):
             debug(json_msg, 3)
             s = socket.socket(socket.AF_UNIX, socket.SOCK_DGRAM)
             s.connect(self.wazuh_queue)
-            s.send("{header}{msg}".format(header=self.msg_header,
-                                          msg=json_msg if dump_json else msg).encode())
+
+            encoded_msg = "{header}{msg}".format(header=self.msg_header,
+                                                 msg=json_msg if dump_json else msg).encode()
+
+            # Logs warning if event is bigger than max size
+            if len(encoded_msg) > MAX_EVENT_SIZE:
```

Comment:
The AWS module imports the entire utils file. Therefore, this line will throw an error.
```suggestion
            if len(encoded_msg) > utils.MAX_EVENT_SIZE:
```

Reply:
Fixed in commit 113469afbaff8582af4db430587402c60c697277:

https://github.com/wazuh/wazuh/blob/113469afbaff8582af4db430587402c60c697277/wodles/aws/aws_s3.py#L519-L520